packages/components/src/public/SchemaQuery.ts

```diff
@@ -104,8 +104,8 @@ export class SchemaQuery extends Record({
         return false;
     }
 
-    getKey(): string {
-        return resolveKey(this.schemaName, this.queryName, this.viewName);
+    getKey(includeViewName = true): string {
+        return resolveKey(this.schemaName, this.queryName, includeViewName ? this.viewName : undefined);
```

Comment:
Looks like the `viewName` was added to the key earlier this year by @labkey-susanh (as part of the PR for issue 46137 and 46121). Just want to make sure this change doesn't regress anything in that area.

Reply:
The only place this is being disabled (the use of viewName in the key) is for QueryInfo cache keys. We do not differentiate QueryInfo's based on view name so this was causing multiple requests for the same QueryInfo.

Everywhere else that uses `SchemaQuery.getKey()` will continue to operate with the view name as intended with Susan's work.

Reply:
Perfect. That makes sense. I forgot that the QueryInfo object doesn't actually differ per view name.

Reply:
This also happens to address [Issue 46749](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=46749) (opened yesterday - thanks for linking me to it @cnathe).